### ISSUE TITLE
Can't run vagrant up (vagrant works for other boxes)

### ISSUE THREAD

**Issue by HenryMarshall** [freelawproject/freelawmachine#23]
When I run `vagrant up` with the [official Vagrantfile](https://github.com/freelawproject/freelawmachine/blob/master/Vagrantfile), I get the following error:

> Timed out while waiting for the machine to boot. This means that
> Vagrant was unable to communicate with the guest machine within
> the configured ("config.vm.boot_timeout" value) time period.

I tried increasing the timeout to 10 minutes by adding `config.vm.boot_timeout = 600` inside the `Vagrant.configure(2)` block. This did not fix the problem.

I have no troubles when using a vanilla [ubuntu 16.04](https://atlas.hashicorp.com/ubuntu/boxes/xenial64) vagrant box.

I am running

Vagrant 1.9.5
VirtualBox 5.1.22r115126
Kubuntu 16.04

The docs specify Vagrant 1.8.5, but I did not see a way to get an old `deb` installer from vagrant.

----

Below you will find a complete copy of what is printed when I post.

$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'freelawproject/freelawbox64' is up to date...
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 8000 (guest) => 8000 (host) (adapter 1)
    default: 8081 (guest) => 8081 (host) (adapter 1)
    default: 8983 (guest) => 8983 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
Timed out while waiting for the machine to boot. This means that
Vagrant was unable to communicate with the guest machine within
the configured ("config.vm.boot_timeout" value) time period.

If you look above, you should be able to see the error(s) that
Vagrant had when attempting to connect to the machine. These errors
are usually good hints as to what may be wrong.

If you're using a custom box, make sure that networking is properly
working and you're able to connect to the machine. It is a common
problem that networking isn't setup properly in these boxes.
Verify that authentication configurations are also setup properly,
as well.

If the box appears to be booting properly, you may want to increase
the timeout ("config.vm.boot_timeout") value.

$ vagrant ssh
ssh_exchange_identification: read: Connection reset by peer
```

**Comment by mlissner** [comment]
Any ideas, @voutilad ?

**Comment by voutilad** [comment]
I'll take a look and report back.

**Comment by voutilad** [comment]
Hmm, it seems to work fine on macOS Sierra using Vagrant 1.9.5. and VirtualBox 5.1.22.

Here's all I did:

1. `git clone https://github.com/freelawproject/freelawmachine`
2. `cd freelawmachine`
3. `vagrant up`

@HenryMarshall what does `vagrant box list | grep freelawproject` return on your machine? You should see something like:
```bash
$ vagrant box list | grep freelawproject
freelawproject/freelawbox64       (virtualbox, 2.1.0)
```
If it's not similar (e.g. v2.1.0) I recommend deleting the box you have, re-cloning the repo, and trying another `vagrant up`.

**Comment by HenryMarshall** [comment]
**tl;dr** It works now, thanks! 

When I ran `vagrant box list | grep freelawproject`, I did get the (exact) expected result. I ran `vagrant box remove freelawproject/freelawbox64`, cloned down the repo, and ran `vagrant up`. This time everything worked perfectly.

The only difference in strategy was that the first time, I simply downloaded the Vagrantfile and the second time I cloned the whole repo. From the README:

> Either clone this project using git or download a copy of the Vagrantfile. Copy to a wherever you want to have CourtListener available.

When I tried to replicate the failure, by copying the Vagrantfile into a temporary directory. I don't have much experience with vagrant/VMs, so I don't know if it might be related to my earlier successful install.

**Comment by voutilad** [comment]
Thanks for the feedback and glad to know you're functional!

I'll see if there's an issue with using just the Vagrantfile. It's very possible there's a bug. I do know Github had some service outages and part of the provisioning is to pull files from Github if the whole repo isn't cloned.
